### PR TITLE
Updated docs to add write permissions to local storage

### PR DESCRIPTION
**Pull request by jonathankingfc**
### Description of Changes

- Added command to add write permissions for Quay container uid on host

## Reviewer Checklist
@thomasmckay 

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

### CODE REVIEW

**Comment by jonathankingfc** [comment]
### UPDATED Description of Changes

- Added command to add write permissions for Quay container uid on host
- Removed logging calls for "service_key_create" and "service_key_approve"
- Resolved "not-null value" issue with Peewee

## Reviewer Checklist
@josephschorr 

- [ ] It works!
- [ ] Comments provide sufficient explanations for the next contributor
- [ ] Tests cover changes and corner cases
- [ ] Follows Quay syntax patterns and format

File: docs/getting_started.md [hunk 1]
```diff
@@ -116,6 +116,7 @@ While we're here, let's set up a directory to hold image blobs:
 
 ```
 $ mkdir $QUAY/storage
+$ setfacl -m u:1001:-wx $QUAY/storage
```
Comment (josephschorr) [inline docs/getting_started.md:119]: Please remove this change from the PR

**Comment by jonathankingfc** [comment]
@josephschorr @alecmerdler

**Comment by alecmerdler** [comment]
@jonathankingfc You need to run `make black` to format the Python code (`black` needs Python3 to run, so you can't use the same `virtualenv` as Quay).

**Comment by jonathankingfc** [comment]
Fixed PR to just include docs change

**Pull request merged**